bolt/include/bolt/Core/MCPlusBuilder.h

```diff
@@ -1208,6 +1218,14 @@ class MCPlusBuilder {
     return nullptr;
   }
 
+  /// Return MCSymbol that represents a target of this instruction at a given
+  /// operand number \p OpNum. If there's no symbol associated with
+  /// the operand - return nullptr.
+  virtual MCSymbol *getTargetSymbol(MCInst &Inst, unsigned OpNum = 0) const {
```

Comment:
Is this needed?
Looks like it's only used in. Can that be const?

```
OS << "\tjit\t" << MIB->getTargetSymbol(Instruction)->getName()
       << " # ID: " << DynamicID;
```

Reply:
Not needed anymore.